src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate.java

```diff
@@ -330,6 +332,9 @@ public String getDisplayName() {
         }
 
         public FormValidation doCheckName(@QueryParameter String value) {
+            if (StringUtils.isEmpty(value)) {
+                return FormValidation.warning("Container name is mandatory.");
```

Comment:
The image check uses `FormValidation.ok()` for this, but it really doesn't stand out on the page. Maybe someone from UX can take a quick look and make a judgement on what we should use.

Also this warning appears as soon as you add a container template in the form, it doesn't wait for the text box to gain and lose focus.